bot.py

```diff
@@ -87,7 +87,8 @@ def check_and_return(self, value: types.Message) -> types.Location:
             range_ = 500
             response = '반경이 너무 크거나 작아요. 기본값인 500미터로 고정할게요.\n'
     response += '이 메세지의 답변 메세지로 현재 위치를 보내주세요.'
-    sent_message = await bot.send_message(message.chat.id, response, reply_to_message_id=message.message_id)
+    markup = types.ForceReply()
+    sent_message = await bot.send_message(message.chat.id, response, reply_to_message_id=message.message_id, reply_markup= markup)
```

Comment:
Since `markup` variable is just a reference to `types.ForceReply()`, it'll be better to call `types.ForceReply()` directly from `bot.send_message` instead of creating additional variable.